src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -22,14 +22,22 @@ public partial class Socket : IDisposable
     {
         internal const int DefaultCloseTimeout = -1; // NOTE: changing this default is a breaking change.
 
+        private static readonly HashSet<IPAddress> s_wildcardAddresses = new HashSet<IPAddress>
```

Comment:
With only 4 items an array would be cheaper here.

Reply:
Fixed